uaclient/messages/__init__.py

```diff
@@ -942,7 +942,10 @@ class TxtColor:
 This information can be later used for triaging/debugging issues."""
 )
 CLI_COLLECT_LOGS_OUTPUT = t.gettext(
-    "tarball where the logs will be stored. (Defaults to " "./pro_logs.tar.gz)"
+    """\
+tarball where the logs will be stored. (Defaults to ./pro_logs.tar.gz).
+Notice that the file must not already exist in the system, otherwise
```

Comment:
I think this message can stay as-is, we don't need to be explicit about the file existing, because the error message is explicit